### PR TITLE
Added all 4 controls for few more GCP resources like bigtable, dataproc and pubsub

### PR DESCRIPTION
**Pull request by leegin**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: controls/limit.sp [hunk 1]
```diff
@@ -47,7 +47,12 @@ benchmark "limit" {
     control.compute_snapshot_label_limit,
     control.dns_managed_zone_label_limit,
     control.sql_database_instance_label_limit,
-    control.storage_bucket_label_limit
+    control.storage_bucket_label_limit,
+    control.bigtable_instance_label_limit,
+    control.dataproc_cluster_label_limit,
+    control.dataproc_job_label_limit,
+    control.pubsub_subscription_label_limit,
+    control.pubsub_topic_label_limit
```
Comment (misraved) [inline controls/limit.sp:55]: Please sort the controls with the others in the benchmark.

The sorted list should look something as follows:

```    control.bigquery_dataset_label_limit,
    control.bigquery_job_label_limit,
    control.bigquery_table_label_limit,
    control.bigtable_instance_label_limit,
    control.compute_disk_label_limit,
    control.compute_forwarding_rule_label_limit,
    control.compute_image_label_limit,
    control.compute_instance_label_limit,
    control.compute_snapshot_label_limit,
    control.dataproc_cluster_label_limit,
    control.dataproc_job_label_limit,
    control.dns_managed_zone_label_limit,
    control.pubsub_subscription_label_limit,
    control.pubsub_topic_label_limit,
    control.sql_database_instance_label_limit,
    control.storage_bucket_label_limit

File: controls/limit.sp [hunk 2]
```diff
@@ -177,3 +182,48 @@ control "storage_bucket_label_limit" {
     default = var.label_limit
   }
 }
+
+control "bigtable_instance_label_limit" {
+  title       = "Bigtable instances should not exceed label limit"
+  description = "Check if the number of labels on bigtable instance do not exceed the limit."
```
Comment (misraved) [inline controls/limit.sp:188]:
```suggestion
  description = "Check if the number of labels on bigtable instances do not exceed the limit."
```

File: controls/limit.sp [hunk 3]
```diff
@@ -177,3 +182,48 @@ control "storage_bucket_label_limit" {
...
+  sql         = replace(local.limit_sql_location, "__TABLE_NAME__", "gcp_bigtable_instance")
+  param "label_limit" {
+    default = var.label_limit
+  }
+}
+
+control "dataproc_cluster_label_limit" {
+  title       = "Dataproc clusters should not exceed label limit"
+  description = "Check if the number of labels on dataproc cluster do not exceed the limit."
```
Comment (misraved) [inline controls/limit.sp:197]:
```suggestion
  description = "Check if the number of labels on dataproc clusters do not exceed the limit."
```

File: controls/mandatory.sp [hunk 4]
```diff
@@ -51,7 +51,12 @@ benchmark "mandatory" {
     control.compute_snapshot_mandatory,
     control.dns_managed_zone_mandatory,
     control.sql_database_instance_mandatory,
-    control.storage_bucket_mandatory
+    control.storage_bucket_mandatory,
+    control.bigtable_instance_mandatory,
+    control.dataproc_cluster_mandatory,
+    control.dataproc_job_mandatory,
+    control.pubsub_subscription_mandatory,
+    control.pubsub_topic_mandatory
```
Comment (misraved) [inline controls/mandatory.sp:59]: Please make changes as per https://github.com/turbot/steampipe-mod-gcp-labels/pull/18/files#r961570223

File: controls/prohibited.sp [hunk 5]
```diff
@@ -56,7 +56,12 @@ benchmark "prohibited" {
     control.compute_snapshot_prohibited,
     control.dns_managed_zone_prohibited,
     control.sql_database_instance_prohibited,
-    control.storage_bucket_prohibited
+    control.storage_bucket_prohibited,
+    control.bigtable_instance_prohibited,
+    control.dataproc_cluster_prohibited,
+    control.dataproc_job_prohibited,
+    control.pubsub_subscription_prohibited,
+    control.pubsub_topic_prohibited
```
Comment (misraved) [inline controls/prohibited.sp:64]: Please make changes as per https://github.com/turbot/steampipe-mod-gcp-labels/pull/18/files#r961570223

File: controls/mandatory.sp [hunk 6]
```diff
@@ -185,3 +190,48 @@ control "storage_bucket_mandatory" {
     default = var.mandatory_labels
   }
 }
+
+control "bigtable_instance_mandatory" {
+  title       = "Bigtable instance should have mandatory labels"
+  description = "Check if Storage bigtable instance have mandatory labels."
```
Comment (misraved) [inline controls/mandatory.sp:196]:
```suggestion
  description = "Check if Storage bigtable instances have mandatory labels."
```

**Comment by misraved** [comment]
Thanks @leegin for adding the controls across all the benchmarks. All the changes look good and are ready to be published to the hub. Could you please take a look at the summary of the changes that are expected before we merge the PR to the release branch:
- Sort the controls under a benchmark (as mentioned in https://github.com/turbot/steampipe-mod-gcp-labels/pull/18/files#r961570223)
- The titles and descriptions of the controls need to use the plural form of resource types, for instance, `Bigtable instances` instead of `Bigtable instance` and `Pubsub subscriptions` instead of `Pubsub subscription`.

These are just a few minor observations barring which the other changes look great 👍 .

**Comment by leegin** [comment]
> Thanks @leegin for adding the controls across all the benchmarks. All the changes look good and are ready to be published to the hub. Could you please take a look at the summary of the changes that are expected before we merge the PR to the release branch:
> 
> * Sort the controls under a benchmark (as mentioned in https://github.com/turbot/steampipe-mod-gcp-labels/pull/18/files#r961570223)
> * The titles and descriptions of the controls need to use the plural form of resource types, for instance, `Bigtable instances` instead of `Bigtable instance` and `Pubsub subscriptions` instead of `Pubsub subscription`.
> 
> These are just a few minor observations barring which the other changes look great 👍 .

@misraved I have made all the changes suggested. Please review it.

**Pull request merged**